test/parallel/test-stream-duplex-end.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const Duplex = require('stream').Duplex;
+{
+  const stream = new Duplex();
+  assert(stream.allowHalfOpen);
```

Comment:
Can you use `assert.strictEqual()` here.

Reply:
@cjihrig Fixed it. Thank you for your comment.